### ISSUE TITLE
Hipothesys: parliamentarians may request reimbursements for expenses that do not meet the criteria

### ISSUE THREAD

**Issue by luizfzs** [datasciencebr/serenata-de-amor#170]
As I stated in #169, I worry that some parliamentarians may be requesting reimbursements for expenses that do not meet the criteria. As far as I saw, we may already have the data needed for such analysis. I'd love to work on that but I don't have the knowledge on Data Science nor on the project (yet).

I think about it as an easy way to spot illegal requests.

What is your opinion?

**Comment by fgrehm** [comment]
Yeah, while looking at other stuff I found this http://jarbas.datasciencebr.com/#/document_id/5733936 which is a receipt for fuel but is categorized as a congressperson meal

**Comment by jrandson** [comment]
@fgrehm All of these lnks are broken. Bad Gateway. Where I can check for the reimbursements?

**Comment by cuducos** [comment]
> @fgrehm All of these lnks are broken. Bad Gateway. Where I can check for the reimbursements?

I'm sorry @jrandson, we're migrating Jarbas from `jarbas.datascience.com` to `jarbas.serenatadeamor.org`. If you have our datasets, just look for these `document_id`: 5733936 and 5919483.

Otherwise Jarbas will be back online in a hour or so.

**Comment by jrandson** [comment]
Ok

**Comment by cuducos** [comment]
Jarbas is up again.

**Comment by caduvieira** [comment]
Can a senator ask for reimbursement if the company is owing money to the government (Dívida Ativa da União)?
http://www.receita.fazenda.gov.br/Aplicacoes/ATSPO/Certidao/CndConjuntaInter/InformaNICertidao.asp?Tipo=1
